applications/vanilla/views/discussion/tag.php

```diff
@@ -0,0 +1,16 @@
+<?php if (!defined('APPLICATION')) exit(); ?>
+    <h1><?php echo $this->data('Title'); ?></h1>
+<?php
+/** @var Gdn_Form $form */
+$form = $this->Form;
+echo $form->open(['id' => 'DiscussionAddTagForm']);
+echo $form->errors();
+
+echo '<div class="Form-Tags P">';
+    // Tag text box
+    echo $this->Form->label('Tags', 'Tags');
+    echo $this->Form->textBox('Tags', array('data-tags' => ''));
+echo '</div>';
+
+?>
+<?php echo $this->Form->close('Save');
```

Comment:
I personally like it "mininal" but it wouldn't hurt to add a description I guess.

As for the layout, I suggest that you raise an issue about it so that we can review these dialogs since it is identical to the dialog in the dashboard.